src/stores/fileStore.ts

```diff
@@ -5,20 +5,29 @@ import { HL7File, HL7Message } from '../types';
 export interface IFileStore {
   files: HL7File[];
   currentMessage?: HL7Message;
+  file?: HL7File;
```

Comment:
Personally would prefer calling this `currentFile` along to match `currentMessage`.

Reply:
Makes sense @orndorffgrant . Implemented.